build/ical.js

```diff
@@ -1256,12 +1373,18 @@ ICAL.stringify = (function() {
   stringify.component = function(component, designSet) {
     var name = component[0].toUpperCase();
     var result = 'BEGIN:' + name + LINE_ENDING;
-    designSet = designSet || design.getDesignSet(component[0]);
 
     var props = component[1];
     var propIdx = 0;
     var propLen = props.length;
 
+    var designSetName = component[0];
+    if (designSetName === 'vcard' && component[1].length > 0 &&
+            component[1][0][0] === "version" && component[1][0][3] === "3.0") {
```

Comment:
Can you add a quick comment here, noting that the rfc requires VERSION to be the first property?